tensorflow/python/ops/ctc_ops.py

```diff
@@ -180,8 +180,8 @@ def ctc_greedy_decoder(inputs, sequence_length, merge_repeated=True):
   only the first of these is emitted.  The sequence `A B B * B * B` (where '*'
   is the blank label) becomes
 
-    * `A B` if `merge_repeated=True`.
-    * `A B B B B B` if `merge_repeated=False`.
+    * `A B B B` if `merge_repeated=True`.
```

Comment:
No, everything is fine. In those two examples, if you take `argmax` you will obtain:
1. `A * * B` which will transform into `A B`.
2. `B B * B A` which will transform into `B B A`.